package.json

```diff
@@ -371,7 +371,6 @@
     "**/resolve": "1.8.1",
     "**/semver": "7.3.2",
     "**/terser": "5.14.2",
-    "**/xmldom": "github:xmldom/xmldom#c568938641cc1f121cef5b4df80fcfda1e489b6e",
```

Comment:
Nothing depends on this library. Not sure when that changed, but we don't need this.